src/index.ts

```diff
@@ -106,10 +107,22 @@ client.on('messageCreate', async (message) => {
     channel = message.member?.voice.channel;
 
     if (channel) {
-      if (userCommand === 'cmere') {
+      if (userCommand === BotCommands.CMERE) {
         const connection = await connectToChannel(channel);
         connection?.subscribe(audioPlayer);
-      } else if (userCommand === 'gtfo') {
+
+        // Check if playing league
+        if (
+          message.member?.presence &&
+          presenceIndicatesPlayingLeagueOfLegends(message.member.presence) &&
+          message.member.presence.activities[0].state === PresenceState.IN_GAME &&
+          IS_LOL_ANNOUNCER_ENABLED
```

Comment:
This logic is pretty duplicative of the previous PR, open to suggestions on making this more DRY